simtools/production_configuration/generate_simulation_config.py

```diff
@@ -0,0 +1,193 @@
+"""
+Module defines the `SimulationConfig` class.
```

Comment:
The short docstring should describe the functionality of this calls in one line. e.g., `Configures and generates simulation parameters for a specific grid point`